### ISSUE TITLE
Overlay/mouseover box issues

### ISSUE THREAD

**Issue by epi-sam** [damiankorcz/Prism-Theme#19]
I think this is one of the best themes right now.  The interface adjustments are fantastic.

For the recent update, I've noticed issues with overlay boxes in three situations:
1. Preview of a link - text below a certain point is not visible (the displayed file has text all the way to the bottom)
![image](https://user-images.githubusercontent.com/96550052/147771667-f649d830-b62a-428e-a5dd-78eb32cc8eb6.png)

2. Calendar view for the _Reminder_ plugin - the cursor prefers to select text under the calendar, rather than select a date from the calendar.
![image](https://user-images.githubusercontent.com/96550052/147771797-f826de23-f047-4724-8a3d-a73b8529e39b.png)

3. Header tags overlap the header section text when writing internal links
![image](https://user-images.githubusercontent.com/96550052/147771905-4ce4f22a-d403-4cb1-a6aa-3b66ccaf819d.png)

I'm sorry I'm unable to make a screen clip gif to illustrate.  I'm not yet familiar with that process.

Best regards,

**Comment by epi-sam** [comment]
The following plugins are installed, if that helps identify potential conflicts
```
          .
          ..
          calendar
          cm-editor-syntax-highlight-obsidian
          file-explorer-note-count
          highlightr-plugin
          nldates-obsidian
          obsidian-day-planner
          obsidian-editor-shortcuts
          obsidian-hider
          obsidian-minimal-settings
          obsidian-plugin-todo
          obsidian-reminder-plugin
          obsidian-statusbar-pomo
          obsidian-style-settings
          oz-image-plugin
          quick-explorer
          remember-cursor-position
          shortcuts-extender
          table-editor-obsidian
```

**Comment by damiankorcz** [comment]
@epi-sam Thanks for filing the issue. I'm looking into it right now. I've mostly avoided working on additional plugins for the time being so number 2 might have to wait a little unless it's a simple fix. I'll try to resolve these for the next release.

If you encounter any other issues keep them coming!

**Comment by epi-sam** [comment]
Thank you for the amazing theme!  If I see anything major, I'll raise a flag.

**Comment by damiankorcz** [comment]
I couldn't replicate the issue with Number 1 so let me know if I might have fixed it by accident. Number 2 as mentioned before I'll look into addressing in the future. Number 3 should now be fixed in Version 1.2.0.

**Comment by epi-sam** [comment]
Hey the headers fix on point number three is awesome!  Totally understand about the second issue.  For the first, what I see now is no preview at when using a CTRL+mouse-hover while in Editor View.  I do see a preview of the link while in Reading View, although it is truncated, has a two-tone background, and does not preview the full document, as before.  I'll try toggling some other plugins to see if anything shakes looks.  

![image](https://user-images.githubusercontent.com/96550052/147864114-d7f7c129-169a-43cf-9e5e-23eea2dd7385.png)


Again, awesome theme, still my current favorite.  Has the perfect blend of minimalism, but visibility of important information, management of the editor layout, and information density.  Huge props.

**Comment by damiankorcz** [comment]
Hi @epi-sam, the "two-tone" background for note previews should now be addressed in Version 1.5.0. I've also added support for the Reminder as well.

**Comment by epi-sam** [comment]
This looks fantastic, thank you for reaching out!
